File: java/1008.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int numF, numH;
        double salH, salF;
        
        numF = in.nextInt();
        numH = in.nextInt();
        salH = in.nextDouble();
        salF = numH * salH;
        System.out.printf("NUMBER = %d\nSALARY = U$ %.2f\n", numF, salF);
    }
}

File: C/1008.c
#include <stdio.h>

int main(){
    int a, b;
    float c, sal = 0;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%f", &c);
    sal = b * c;
    printf("NUMBER = %d\nSALARY = U$ %.2f\n", a, sal);
    return 0;
}
